### ISSUE TITLE
Release 4.0 tracker

### ISSUE THREAD

**Issue by lresende** [elyra-ai/elyra#3259]
Using this issue to track the must haves before we can start cutting a 4.0 release

- [x] https://github.com/elyra-ai/elyra/pull/3201
- [x] https://github.com/elyra-ai/elyra/pull/3145
- [x] https://github.com/elyra-ai/elyra/pull/3258
- [x] https://github.com/elyra-ai/elyra/pull/3256
- [x] https://github.com/elyra-ai/elyra/pull/3167
- [x] https://github.com/elyra-ai/elyra/pull/3160
- [x] https://github.com/elyra-ai/elyra/pull/3141

**Comment by caponetto** [comment]
@lresende I believe it'd be nice to have the PRs below cherry-picked for the release. The first one is an improvement to build the release artifacts (frontend) using the production scripts, and the other two are bug fixes. I can send a PR for each one of them. Please let me know if I can proceed.

- https://github.com/opendatahub-io/elyra/pull/50
- https://github.com/opendatahub-io/elyra/pull/87
- https://github.com/opendatahub-io/elyra/pull/88

**Comment by lresende** [comment]
@caponetto I tried to cherry-pick some of these with conflicts or other issues. Maybe it would be easier if you submitted a PR with some details on how to validate. We might also need to update some docs for the dev/prod builds.

**Comment by caponetto** [comment]
@lresende I have submitted the PRs. Please take a look at them when you have some time.

**Comment by lresende** [comment]
We should be ready for release after the rc5 released today.

**Comment by lresende** [comment]
https://github.com/elyra-ai/elyra/releases/tag/v4.0.0
